dashboard/app/views/levels/editors/fields/_dsl_fields.html.haml

```diff
@@ -4,7 +4,11 @@
   $lighter_cyan: #a6daed;
   $lightest_cyan: #d9eff7;
 
-  .script_text textarea { width: 100%; height: 100px}
+  .script_text textarea {
+    field-sizing: content;
+    max-width: 100%;
+    min-width: 95%;}
```

Comment:
Need to have it include its own padding in its calculation of `100%`... not all browsers will have the same scrollbar width. I suggest using `border-box` sizing.

```suggestion
    width: 100%;
    box-sizing: border-box;}
```

Reply:
If you want to not allow resizing by width, keeping the `max-width` is reasonable.

Reply:
Wilkie! Thanks for this suggestion :)